src/main/java/com/google/sps/data/Coordinates.java

```diff
@@ -12,6 +12,7 @@
   @SerializedName(value = "Latitude")
   public double lat;
   
+  
```

Comment:
Sneaky extra line

Reply:
theyre so easy to miss if you dont have the git highlighted line changes! o noes